apps/platform-app/src/app/[locale]/(profile)/(components)/MyProfile/ProfileDetails/ProfileDetails.tsx

```diff
@@ -28,6 +28,16 @@ const ProfileDetails = async (props: ProfileDetailsProps) => {
               ))}
             </div>
           </div>
+          {/* <div className={styles.techStack}>
```

Comment:
**nit:** Should it be commented out?

Reply:
I commented it because there is another task in Nerdbord for this case and I totally forgot to delete it here.  I'll use it in second task.